tests/test_datasets/test_dataset.py

```diff
@@ -320,6 +320,14 @@ def test_get_sparse_dataset_rowid_and_ignore_and_target(self):
         self.assertListEqual(categorical, [False] * 19998)
         self.assertEqual(y.shape, (600, ))
 
+    def test_get_sparse_categorical_data_id_395(self):
```

Comment:
This test does not explicitly load the data, so is the function in question actually called?

Reply:
Yeah, my bad. Made it **True**.